lib/client/scm/pgp/sign.ts

```diff
@@ -0,0 +1,41 @@
+import * as openpgp from 'openpgp';
+import { PgpPrivateKeyValidationError } from './errors';
+import type { CreateSignatureOptions, PgpPrivateKey } from './types';
+
+const pgpPrivateKeyBeginBlock = '-----BEGIN PGP PRIVATE KEY BLOCK-----\n';
```

Comment:
@praveen-tripathi-sf, yes.
According the specification it has to be one line break. Additionally it could be a meta information (some tools write `Comment:` and `Version:`) and after another line break there is a content of the key, with ending block.